src/helm-frontend/vite.config.ts

```diff
@@ -17,5 +17,5 @@ export default defineConfig({
   build: {
     outDir: `${__dirname}/../helm/benchmark/static_build`,
   },
-  base: "/helm/",
+  //base: "/helm/" // can't add process.env.HELM_SUITE here with GH pages 
```

Comment:
Could you add some documentation somewhere about how to configure and build the website with these changes e.g. does this have to change to `/helm/latest/` before building?

Reply:
A comment in the code would be sufficient for this, for now.